deploy/helm/charts/values.yaml

```diff
@@ -112,6 +112,7 @@ nfsStorageClass:
 
 nfsServer:
   useClusterIP: "true"
+  imagePullSecret: ""
```

Comment:
lets rename `imagePullSecret` to `imagePullSecrets`

Reply:
If I rename it to this then I should probably also make it an array rather than a single value. I can do that if you want, but I don't see a reason why someone would need more than one image pull secret for the NFS Server since it runs as a single Pod with a single container, so only one image.

If we do choose to make it a list, then I could pass the configuration as a comma-separated list in the environment variable, and split that in the Go code and support adding multiple pull secrets at once like that. But like I said before...I don't think that would be all that helpful. But let me know if that is what you would like me to do. :)